go.mod

```diff
@@ -8,3 +8,5 @@ require (
 	k8s.io/apimachinery v0.16.6
 	k8s.io/client-go v0.16.6
 )
+
+replace github.com/projectriff/stream-client-go => github.com/ericbottard/stream-client-go v0.0.0-20200204150506-7cfb9bf48a59
```

Comment:
Final version will not use this line but instead change the hash on line 6 above, once https://github.com/projectriff/stream-client-go/pull/15 is merged